config/database.yml

```diff
@@ -21,7 +21,9 @@ development:
   pool: 5
   username: safecast
   su_username: safecast
-  password: 
+  password:
+  host: postgresql
```

Comment:
We might want to be careful about adding this. Pretty sure cloud66 uses the checked in database.yml. Should probably back out the change until we verify or move the app to AWS